ray_provider/operators.py

```diff
@@ -173,7 +179,6 @@ def __init__(
         self.xcom_task_key = xcom_task_key
         self.dashboard_url: str | None = None
         self.job_id = ""
-        self.terminal_states = {JobStatus.SUCCEEDED, JobStatus.STOPPED, JobStatus.FAILED}
```

Comment:
I'm not sure if someone uses this property directly, but Shall we add a small note in the changelog about this property removal?

Reply:
Nice catch, I'll add this to the release notes, @pankajastro . I added a note to #77 so we don't forget about it